<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Visión Global</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>611</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="FrameLabel" name="imageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>102</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Capture Rate:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="captureRateLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Processing Rate:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="processingRateLabel">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Posición</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mouseCursorRealPosLabel">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Cursor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FrameLabel" name="mouseCursorPosLabel">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionEditCamara1"/>
    <addaction name="actionEditCamara2"/>
    <addaction name="separator"/>
    <addaction name="actionEditColor"/>
    <addaction name="separator"/>
    <addaction name="actionEditPatch"/>
    <addaction name="separator"/>
    <addaction name="actionDimensionsField"/>
   </widget>
   <widget class="QMenu" name="menuAcerca">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCargar_Configuraci_n_Global"/>
    <addaction name="actionGuardar_Configuraci_n_Global"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <widget class="QMenu" name="menuCalibrate">
     <property name="title">
      <string>Calibrar cámara</string>
     </property>
     <addaction name="actionCalibraMono"/>
     <addaction name="actionCalibraStereo"/>
    </widget>
    <widget class="QMenu" name="menuSearchPatch">
     <property name="title">
      <string>Seguir objeto</string>
     </property>
     <addaction name="actionSearchPatch1"/>
     <addaction name="actionSearchPatch2"/>
     <addaction name="actionSearchPatch3"/>
     <addaction name="separator"/>
     <addaction name="actionSearchBall1"/>
    </widget>
    <widget class="QMenu" name="menuTraining">
     <property name="title">
      <string>Training</string>
     </property>
     <addaction name="actionBayes"/>
    </widget>
    <addaction name="menuCalibrate"/>
    <addaction name="separator"/>
    <addaction name="menuSearchPatch"/>
    <addaction name="menuTraining"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuServidor">
    <property name="title">
     <string>Servidor</string>
    </property>
    <addaction name="actionIniciar"/>
    <addaction name="actionDetener"/>
    <addaction name="actionIniciarTrackingGlobal"/>
    <addaction name="actionDetenerTrackingGlobal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuServidor"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAcerca"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOne">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionTwo">
   <property name="text">
    <string>two</string>
   </property>
  </action>
  <action name="actionThree">
   <property name="text">
    <string>three</string>
   </property>
  </action>
  <action name="actionFour">
   <property name="text">
    <string>four</string>
   </property>
  </action>
  <action name="actionFive">
   <property name="text">
    <string>five</string>
   </property>
  </action>
  <action name="actionOne_2">
   <property name="text">
    <string>one</string>
   </property>
  </action>
  <action name="actionTwo_2">
   <property name="text">
    <string>two</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../unc/Investigaciones/vision_global/qt4/qt-book/chap04/spreadsheet/images/find.png</normaloff>../../../../../unc/Investigaciones/vision_global/qt4/qt-book/chap04/spreadsheet/images/find.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de ...</string>
   </property>
   <property name="statusTip">
    <string>Muestra información del sistema</string>
   </property>
  </action>
  <action name="actionC_mara">
   <property name="text">
    <string>Cámara</string>
   </property>
  </action>
  <action name="actionNueva">
   <property name="text">
    <string>Nueva Cámara</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar Configuración desde Archivo</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar Configuración de Cámara</string>
   </property>
  </action>
  <action name="actionRobots">
   <property name="text">
    <string>Robots</string>
   </property>
  </action>
  <action name="actionCancha">
   <property name="text">
    <string>Cancha</string>
   </property>
  </action>
  <action name="actionEquipo">
   <property name="text">
    <string>Equipo</string>
   </property>
  </action>
  <action name="actionCampeonato">
   <property name="text">
    <string>Campeonato</string>
   </property>
  </action>
  <action name="actionNueva_Configuraci_n_Global">
   <property name="text">
    <string>Nueva Configuración Global</string>
   </property>
  </action>
  <action name="actionCargar_Configuraci_n_Global">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../unc/Investigaciones/vision_global/qt4/qt-book/chap04/spreadsheet/images/find.png</normaloff>../../../../../unc/Investigaciones/vision_global/qt4/qt-book/chap04/spreadsheet/images/find.png</iconset>
   </property>
   <property name="text">
    <string>Cargar Configuración Global</string>
   </property>
  </action>
  <action name="actionGuardar_Configuraci_n_Global">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Guardar Configuración Global</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionCargar_2">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionGuardar_2">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionCalibraMono">
   <property name="text">
    <string>Mono</string>
   </property>
  </action>
  <action name="actionCalibraStereo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stereo</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionNueva_2">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionCargar_3">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionGuardar_3">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionNueva_3">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionCargar_4">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionConfigurar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionNueva_4">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionCargar_5">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionGuardar_4">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionFormato">
   <property name="text">
    <string>Formato</string>
   </property>
  </action>
  <action name="actionSearchPatch1">
   <property name="text">
    <string>Parche1</string>
   </property>
  </action>
  <action name="actionSearchPatch2">
   <property name="text">
    <string>Parche2</string>
   </property>
  </action>
  <action name="actionSearchPatch3">
   <property name="text">
    <string>Parche3</string>
   </property>
  </action>
  <action name="actionSearchBall1">
   <property name="text">
    <string>Pelota</string>
   </property>
  </action>
  <action name="actionSistema_HUE">
   <property name="text">
    <string>HUE</string>
   </property>
  </action>
  <action name="actionPelota_2">
   <property name="text">
    <string>Pelota</string>
   </property>
  </action>
  <action name="actionEditCamara2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cámara 2</string>
   </property>
  </action>
  <action name="actionEditColor">
   <property name="text">
    <string>Colores</string>
   </property>
  </action>
  <action name="actionEditPatch">
   <property name="text">
    <string>Parches</string>
   </property>
  </action>
  <action name="actionGuardar_5">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionConfigurar_par_metros_intr_nsecos">
   <property name="text">
    <string>Configurar parámetros intrínsecos</string>
   </property>
  </action>
  <action name="actionCargar_6">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionGuardar_6">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionIngresar_par_metros">
   <property name="text">
    <string>Ingresar parámetros</string>
   </property>
  </action>
  <action name="actionBayes">
   <property name="text">
    <string>Bayes</string>
   </property>
  </action>
  <action name="actionEditCamara1">
   <property name="text">
    <string>Cámara 1</string>
   </property>
   <property name="statusTip">
    <string>Edición de los parámetros de la Cámara 1</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
  <action name="actionDetener">
   <property name="text">
    <string>Detener</string>
   </property>
  </action>
  <action name="actionIniciarTrackingGlobal">
   <property name="text">
    <string>Iniciar Tracking Global</string>
   </property>
  </action>
  <action name="actionDetenerTrackingGlobal">
   <property name="text">
    <string>Detener Tracking Global</string>
   </property>
  </action>
  <action name="actionDimensionsField">
   <property name="text">
    <string>Dimensiones de la Cancha</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameLabel</class>
   <extends>QLabel</extends>
   <header location="global">FrameLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
